pkg/semodule/policycoreutils/policycoreutils.go

```diff
@@ -0,0 +1,73 @@
+//go:build policycoreutils
+// +build policycoreutils
+
+package policycoreutils
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/containers/selinuxd/pkg/semodule/seiface"
+	"github.com/go-logr/logr"
+)
+
+type SEModulePcuHandler struct {
+	logger logr.Logger
+}
+
+// Ensure that the test handler implements the Handler interface
+var _ seiface.Handler = &SEModulePcuHandler{}
+
+func runSeModule(opFlag, policyArg string) (string, error) {
```

Comment:
you could make this more general by using variable arguments. e.g.

```
func runSeModule(args ...string) (string, error) {
```

Reply:
Done